ray-operator/controllers/ray/common/pod.go

```diff
@@ -264,11 +263,17 @@ func DefaultWorkerPodTemplate(instance rayv1alpha1.RayCluster, workerSpec rayv1a
 	return podTemplate
 }
 
-func initLivenessProbeHandler(probe *v1.Probe, rayNodeType rayv1alpha1.RayNodeType) {
-	if probe.Exec == nil {
-		// we only create the probe if user did not specify any.
```

Comment:
I think this comment is useful and doesn't need to be removed: "// we only create the probe if user did not specify any."

Reply:
Update: https://github.com/ray-project/kuberay/pull/1364/commits/2e2661e405fccf525c1f422c7056df3f687bc0eb